backend/EkoFunkcje/EkoFunkcje/Utils/Exceptions/GoogleResponseWrongAddressException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+
+namespace EkoFunkcje.Utils.Exceptions
+{
+    public class GoogleResponseWrongAddressException : BaseException
+    {
+        public GoogleResponseWrongAddressException(IEnumerable<string> validationErrors) : base(string.Join(';', validationErrors), LogLevel.Error)
```

Comment:
Takze ze względu na front-end, proponuję separator w Join ustawic na `\n` zamiast `;`. Wtedy będziemy mogli wyswietlic jeden pod drugim dosc latwo :)

Reply:
Cofam powyższy komentarz o separatorze. Przy próbie przemyslenia implementacji wyswietlania błędów na frontendzie wyszlo mi, że obecnie API będzie działać niespójnie - w przpadku walidacji pól interwencji zwracana jest lista błędów jako odpowiedz w JSON, a w przypadku walidacji Googlowego adresu zwracamy zwykły string z separatorami. Wydaje mi się, że powinnismy to ujednolicić w obu przypadkach - osobiście pozostałbym przy liście zwracanej jako JSON.

Reply:
Dodałem serializację do Jsona